src/Datasource/Connection.php

```diff
@@ -201,6 +201,6 @@ protected function _log($context)
         $data = json_encode($logData, JSON_PRETTY_PRINT);
         $loggedQuery = new LoggedQuery();
         $loggedQuery->query = $data;
-        $this->_logger->log('debug', $loggedQuery);
```

Comment:
What about the cases where an `ElasticaLog` instance is used? That object needs the level.